src/tests/ftest/soak/soak.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/python
```

Comment:
Does soak deserve its own directory under src/tests ? Maybe the intent is mostly to share some common code with avocado?